typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -291,32 +290,31 @@ export abstract class HyperlaneDeployer<
   protected async configureHook<C extends Ownable>(
     chain: ChainName,
     contract: C,
-    targetHook: IPostDispatchHook,
+    config: HookConfig,
     getHook: (contract: C) => Promise<Address>,
     setHook: (contract: C, hook: Address) => Promise<PopulatedTransaction>,
   ): Promise<void> {
+    if (typeof config !== 'string') {
```

Comment:
😢

Reply:
punting this until https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/3773